Rakefile

```diff
@@ -3,10 +3,9 @@ require 'bundler/setup'
 begin
   require 'rspec/core/rake_task'
 
-  APP_RAKEFILE = File.expand_path("../spec/manageiq/Rakefile", __FILE__)
+  APP_RAKEFILE = File.expand_path('spec/manageiq/Rakefile', __dir__)
   load 'rails/tasks/engine.rake'
   load 'rails/tasks/statistics.rake'
-rescue LoadError
```

Comment:
You have to rescue LoadError here.  Again, please don't modify things that are generated from core.  If you want to change this across the board, this would probably be better modifying in core in the template.